sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java

```diff
@@ -17,6 +17,8 @@
 
 public class Configs {
     private static final Logger logger = LoggerFactory.getLogger(Configs.class);
+    public static final int CPU_CNT = Runtime.getRuntime().availableProcessors();
```

Comment:
To be consistent with other constants in Configs, we define them as `private static final ...` and then provide a getter to get them. 
We should make `CPU_CNT` consistent with other config constants in this class.

Reply:
In fact I just noticed it was `private static final int CPU_CNT = Runtime.getRuntime().availableProcessors();` 
We should change it back to private.

Reply:
added getCPUCnt done.